### PR TITLE
WT-2817 Add wtperf conf to stress test checkpoints with updates

### PR DESCRIPTION
**Pull request by sulabhM**

### CODE REVIEW

**Comment by sulabhM** [comment]
@daveh86 : Can you please review this change

**Comment by daveh86** [comment]
This LGTM

**Comment by agorrod** [comment]
@sulabhM The naming convention for wtperf configurations is to use a ``-`` instead of an ``_``.

File: bench/wtperf/runners/checkpoint_stress.wtperf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Enough data to fill the cache. 100 million 1k records results in two ~6GB
+# tables
+icount=100000000
+create=true
+compression="snappy"
+populate_threads=1
+checkpoint_interval=60
+checkpoint_threads=1
+report_interval=5
+run_time=600
```
Comment (agorrod) [inline bench/wtperf/runners/checkpoint_stress.wtperf]: Most configurations that are in the code base use a shorter runtime than this - I'm inclined to add a comment explaining that the longer runtime is important to ensure checkpoints are completing.

**Comment by agorrod** [comment]
autobuild: retest this please

**Pull request merged**